tensorzero-core/src/embeddings.rs

```diff
@@ -48,7 +48,8 @@ impl ShorthandModelConfig for EmbeddingModelConfig {
         let model_name = model_name.to_string();
         let provider_config = match provider_type {
             "openai" => {
-                EmbeddingProviderConfig::OpenAI(OpenAIProvider::new(model_name, None, None)?)
+                // TODO - decide how to handle the responses api for shorthand models
```

Comment:
do you mind opening an issue for this as well? the comment is not needed in the embeddings file though